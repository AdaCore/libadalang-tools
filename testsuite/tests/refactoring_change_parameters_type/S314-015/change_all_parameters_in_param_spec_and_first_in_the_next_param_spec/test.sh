change_parameters_type -P test.gpr --source=src/my_lib.ads --start-line=2 --start-column=21 --end-line=3 --end-column=22 --new-parameter-type=Boolean
