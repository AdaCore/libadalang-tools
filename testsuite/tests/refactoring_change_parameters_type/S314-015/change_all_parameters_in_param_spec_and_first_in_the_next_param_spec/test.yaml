title: change_all_parameters_in_param_spec_and_first_in_the_next_param_spec

description: refactoring change_parameters_type test

driver: shell_script
