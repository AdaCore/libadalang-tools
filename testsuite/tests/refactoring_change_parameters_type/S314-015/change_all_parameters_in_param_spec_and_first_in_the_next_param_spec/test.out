my_lib.adb
2:28-2:35 Boolean
3:25-3:30 Boolean
my_lib.ads
2:28-2:35 Boolean
3:25-3:30 Boolean

